proj_utils/Parser.py

```diff
@@ -0,0 +1,75 @@
+from time import sleep
+
+from models.Payload import Payload
+import os
+import json
+
+
+class Parser:
+    def __init__(self, payload: Payload):
```

Comment:
this is a weird way of implementing this. you create a new parser for each message. this is not very efficient